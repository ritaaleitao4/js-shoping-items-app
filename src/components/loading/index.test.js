import Loading from '.';

describe('loading component', () => {
    it('it renders correctly', () => {
        expect(Loading()).toMatchSnapshot();
    });
});