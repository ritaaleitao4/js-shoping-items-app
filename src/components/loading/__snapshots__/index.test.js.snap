// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading component it renders correctly 1`] = `
<div class="loading">
  <img src="[object Object]"
       alt="spinner"
  >
</div>
`;
